### PR TITLE
Add possibility to add group members

### PR DESCRIPTION
**Pull request by doskachok**
Reimplementation of #34 after changes on the develop branch.

### CODE REVIEW

**Comment by AwesomeFlax** [comment]
Can you please adjust user identifiers (email in this case) to the left, as we have group headers (avatar, small space, name, all adjusted to left)?
 image

**Comment by AwesomeFlax** [comment]
In future I would like to extend this added identification, provided email should be secondary information, as a primary one we'll use username or name, if it's given.
But that's ok for now, I'll create an issue for that an describe how it should look like

**Comment by AwesomeFlax** [comment]
Also, this table isn't very obvious by itself, it might use some header or some hint texts, as we did in transactions list, at least for quotas, 0$ might mean way too many things in this case:  how much you paid, how much you own, etc
image
image

**Comment by doskachok** [comment]
> Also, this table isn't very obvious by itself, it might use some header or some hint texts, as we did in transactions list, at least for quotas, 0$ might mean way too many things in this case: how much you paid, how much you own, etc image image

At first we need to decide if we need this information in this place at all. The only reason is that I can potentially see, is when administrator wants to delete member. In this case we can just show if member's balance is 0.

Anyway I would prefer it to be done in a separate PR.

File: src/features/groups/Members/AddMemberModal/index.styled.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import { ModalBody } from '../../../../components/Modal';
```
Comment (AwesomeFlax) [inline src/features/groups/Members/AddMemberModal/index.styled.ts]: isn't that too many `../`, huh? :)

Comment (doskachok) [reply src/features/groups/Members/AddMemberModal/index.styled.ts]: Fixed in e02c021727f336a84c732171816e834c95ca5779

File: src/features/groups/Members/AddMemberModal/UserRow/index.styled.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import { TextRegular } from 'components/Text';
+import styled from 'styled-components';
+
+export const Wrapper = styled(Row)`
+  align-items: center;
+  gap: 0.5rem;
+`;
+
+export const UserNameOrEmail = styled(TextRegular)`
+    color: ${props => props.theme.colors.secondary};
```
Comment (apximax) [inline src/features/groups/Members/AddMemberModal/UserRow/index.styled.ts]: weird indent

File: src/features/groups/Members/AddMemberModal/index.styled.ts
Comment (apximax) [inline src/features/groups/Members/AddMemberModal/index.styled.ts:1]: You forgot 4 ';' :-)

File: src/features/groups/Members/MemberRow/index.styled.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+export const Wrapper = styled(Row) <IWrapperProps>`
+  justify-content: flex-start;
+  align-items: center;
+  gap: 0.5rem;
+  padding: 0.5rem;
+ ${props => props.isSelected && css`
+    background: ${props => props.theme.colors.selectedItem};
+  `};
```
Comment (apximax) [inline src/features/groups/Members/MemberRow/index.styled.ts]: weird indent ('$' sign)

File: src/features/groups/Members/index.styled.ts [hunk 4]
```diff
@@ -1,7 +1,15 @@
...
 
 export const ContentWrapper = styled(Column)`
   padding: 1rem;
   flex: 1;
 `;
+
+export const RemoveBtn = styled(ButtonStyled)`
+  background: ${props => props.theme.colors.button.remove};
+  &:active {
+      background: ${props => props.theme.colors.button.removeFocused};
```
Comment (apximax) [inline src/features/groups/Members/index.styled.ts]: hmmmmm, why 4 whitespaces instead of 2 ?..

**Review by apximax — DISMISSED** [review]

**Pull request merged**